### DISCUSSION TITLE
MalwareBytes alerting to vtf.createit.pl in path:  "W:\laragon\bin\apache\httpd-2.4.54-win64-VS16\bin\httpd.exe"

### DISCUSSION THREAD

**Discussion by Sheldon-Monk** [leokhoa/laragon#537] [General]
For some reason, MalwareBytes flags the URL vtf.createit.pl, whenever we open any development sites running under laragon. It points the httpd.exe file in path:  w;|laragon\bin\apache\httpd-2.4.54-win64-vs16\bin\httpd.exe.  MalwareBytes states it's a "Malware.Ransom.Agent.Generic.  See: https://www.malwarebytes.com/blog/detections/malware-ransom-agent-generic. Obviously, this is a problem.  Anyone else having this issue???
We are using Laragon Full 6.0 220916 php-7.4.33-nts-Win32-vc15-x64 (NTS) on local IP 192.168.50.60.
Really would like to get this remedied.

**Comment by Sheldon-Monk** [comment]
Here is another alert from the same local site running on laragon.  See attachment:

![MalwareBytes Alert2](https://github.com/leokhoa/laragon/assets/34046299/9842ef6d-ac76-41b8-bbfe-f9f1d4986740)

When I try the IP address in the alert, Malware again blocks it.

**Comment by MackSix** [comment]
VirusTotal has MalwareBytes and it did not detect any problems with httpd.exe.

https://www.virustotal.com/gui/file/9cece6414a4d75476811327d51b107925af3f7d2c5bff5ef0f8ad0ebcfd24759

It shows nothing for php-cgi.exe in your version either.

https://www.virustotal.com/gui/file/c089d94d90eb3efaabd1200c17dc232000902df88247f98da7c00df3ad344319

Obviously your version of Malwarebytes detection is wrong.

**Comment by Sheldon-Monk** [comment]
Don't know anything about VirusTotal; especially, you didn't state which version of MalwareBytes is in it.  Bottom line, the current version of the REAL Malwarebytes reports (see the image I included), the file is suspect.  This has done so on three different installs on three separate workstations, on three different networks that we tried it on.

**Reply by MackSix** [reply]
VirusTotal has over 40 virus/malware checkers and none detected anything in the two files from the offical Apache and PHP downloads. The REAL Malwarebytes Premium 4.5.33 is the current version and detects nothing on my Windows 11 desktop.

**Reply by Sheldon-Monk** [reply]
Good for you.  Doesn't matter.  We still had a hit, and it wasn't for nott.

**Reply by MackSix** [reply]
It is not a laragon problem. It's a problem with what is being served from the domain or Malwarebyte Web Protection problem.
